src/transport/pci.rs

```diff
@@ -366,6 +366,18 @@ impl Transport for PciTransport {
     }
 }
 
+impl PciTransport {
+    /// Get the MSI-X vector for queue.
+    pub fn get_queue_msix_vector(&self) -> u16 {
```

Comment:
Don't you need to write to `queue_select` first to choose which queue you're getting the MSI-X vector for?